lib/plugins/aws/package/compile/events/apiGateway/index.js

```diff
@@ -68,10 +68,24 @@ class AwsCompileApigEvents {
           .then(this.compileStage);
       },
 
+      // TODO should be removed once AWS fixes the CloudFormation problems using a separate Stage
+      'after:deploy:deploy': () => {
+        const updateStage = require('./lib/hack/updateStage').updateStage; // eslint-disable-line
```

Comment:
Which eslint rule do we disable? 

I think it's good to be specific, it also prevents turning off other valuable rules, which e.g. may pick a typo, or other significant issue